### PR TITLE
add changeAuthor function

### PR DESCRIPTION
**Pull request by Akira-Taniguchi**
## Proposed Changes

Updated property abi to match dip23.
and changeAothor function.

Merge this into mian and then merge it.

### CODE REVIEW

**Review by aggre — APPROVED** [review]

**Comment by aggre** [comment]
@Akira-Taniguchi Good!

File: lib/property/changeAuthor.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export type CreateChangeAuthorCaller = (
+	contract: Contract,
+	client: Web3
+) => (nextOwther: string) => Promise<boolean>
+
+export const createChangeAuthorCaller: CreateChangeAuthorCaller = (
+	contract: Contract,
+	client: Web3
+) => async (nextOwther: string) =>
```
Comment (aggre) [inline lib/property/changeAuthor.ts:14]: @Akira-Taniguchi  Is this typo? `nextOwther`

Comment (Akira-Taniguchi) [reply lib/property/changeAuthor.ts:14]: yes, this is typo.
I will update it now.

Comment (Akira-Taniguchi) [reply lib/property/changeAuthor.ts]: I fix it now. @aggre

**Review by aggre — APPROVED** [review]
Good!

**Comment by aggre** [comment]
@Akira-Taniguchi I think now mergeable it.

**Pull request merged**